src/pages/getting-started/installing-livechat/index.mdx

```diff
@@ -11,12 +11,12 @@ desc: "Installing LiveChat is a breeze and takes only a few minutes. Follow the
 
 Installing LiveChat is simple and only takes a few minutes. Once installed, you’ll be able to talk to your customer’s right away.
 
-1. Log in to your LiveChat account and go to Settings &rarr; Channels &rarr; [Chat widget](https://my.livechatinc.com/settings/code).
-2. Then, click on Copy code.
+1. Log in to your LiveChat account and go to Settings &rarr; Channels &rarr; [Website](https://my.livechatinc.com/settings/code).
+2. Click on Install LiveChat code manually and then Copy code.
 3. Paste the code to your website’s source code before the `</body>` tag.
 
 Once you’ve successfully added LiveChat code to your website, refresh the page. You should see a chat widget in the bottom right corner.
-Chat widget installed on a website
+Chat widget installed on a website.
```

Comment:
I would change that sentence slightly as it sounds unnatural to me 😅  I would write: A Chat widget is now installed on a website.
@ale-kacperczyk wdyt? 🤔

Reply:
Definitely. Just keep in mind one thing:

- the Chat Widget (the LiveChat Chat Widget, the one you have when you install LiveChat's code)
- a chat widget - any widget for chatting, for example, a custom widget you can build using our [Customer SDK](https://developers.livechat.com/docs/extending-chat-widget/customer-sdk)

So this one should be `The Chat Widget is now installed...`